web/map.html

```diff
@@ -16,6 +16,7 @@
 		<title>TF2 Stats</title>
 		<link rel="stylesheet" href="tachyons.min.css">
 		<link rel="stylesheet" href="master.css">
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.4/Chart.min.js" integrity="sha512-d9xgZrVZpmmQlfonhQUvTR7lMPtO7NkZMkA0ABN3PHCbKA5nqylQ/yWlFAyY6hYgdF1Qh6nYiuADWwKB4C2WSw==" crossorigin="anonymous"></script>

```

Comment:
Using CDN services is a bad practice on the long run

Reply:
Download it locally and commit @Lince99

Reply:
ok Fine, I'll do it myself

see: https://github.com/Oples/tf2-stats/commit/330ae7ba270d8c1325fd68dc339032eb96295e30